tests/testthat/test-utils.R

```diff
@@ -0,0 +1,16 @@
+context("utils.R")
+
+test_that("is_positive_integer returns correct boolean", {
```

Comment:
Might be worth adding a test case for `Inf`

Reply:
Sure!